runtime/stringbuf.c

```diff
@@ -219,7 +219,7 @@ rsCStrConstructFromCStr(cstr_t **const ppThis, const cstr_t *const pFrom)
 
 void rsCStrDestruct(cstr_t **const ppThis)
 {
-	free((*ppThis)->pBuf);
+	if ((*ppThis)->pBuf) free((*ppThis)->pBuf);
```

Comment:
I think there is no need to check the condition. If `(*ppThis)->pBuf` is `NULL`, then no operation is performed by the `free()` function as per `man 3 free`.